src/components/Join us/JoinUs.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import Link from "next/link";
+import Carousel from 'react-bootstrap/Carousel'; 
+
+
+
+
+const JoinUs: React.FC = () => {
```

Comment:
Use function declaration rather than anonymous functions (just a convention that we're going for)
```suggestion
export default function JoinUs() {
}
```